io.catenax.shopfloor_information.provide_production_forecast/1.0.0/ProvideProductionForecast.ttl

```diff
@@ -0,0 +1,128 @@
+   samm:events ( ) .
+
+:productionForecastResponse a samm:Property ;
+   samm:preferredName "Production Forecast Response"@en ;
+   samm:description "The concrete information about a production forecast"@en ;
+   samm:characteristic :ProductionForecastCharacteristic .
+
+:ProductionForecastCharacteristic a samm:Characteristic ;
+   samm:preferredName "Production Forecast Characteristic"@en ;
+   samm:description "All Data that is related to a production forecast"@en ;
+   samm:dataType :ProductionForecastItem .
+
+:ProductionForecastItem a samm:Entity ;
+   samm:preferredName "Production Forecast Item"@en ;
+   samm:description "Datamodel of the concrete production forecast for one forecast item"@en ;
+   samm:properties ( :listOfForecastItems [ samm:property :iterationNo; samm:optional true ] ext-types:communicationMode ext-types:versionDataModel ) .
+
+:listOfForecastItems a samm:Property ;
+   samm:preferredName "List Of Forecast Items"@en ;
+   samm:description "List of ForecastItems matching the items to an order"@en ;
+   samm:characteristic :ForecastItemList .
+
+:iterationNo a samm:Property ;
+   samm:preferredName "IterationNo"@en ;
+   samm:description "Only set in CommunicationMode == \"notification/cyclic\" to be able to check the order of the notifications. \n\nNot required for communication mode = \"synchronous\""@en ;
+   samm:characteristic ext-types:IntegerValueCharacteristic ;
+   samm:exampleValue 6 .
+
+:ForecastItemList a samm-c:List ;
+   samm:preferredName "Forecast Item List"@en ;
+   samm:description "List with the forecast items"@en ;
+   samm:dataType :ForecastItem .
+
+:ForecastItem a samm:Entity ;
+   samm:preferredName "ForecastItem"@en ;
+   samm:description "a ForecastItem entry for the requested order"@en ;
+   samm:properties ( :positionId :productionForecast :precisionOfForecast :productionStatus [ samm:property :reasons4Delay; samm:optional true ] :returnCode :forecastDate ) .
+
+:positionId a samm:Property ;
+   samm:preferredName "PositionId"@en ;
+   samm:description "Identifier of a position of an order"@en ;
+   samm:characteristic ext-header:UuidCharacteristic ;
+   samm:exampleValue "00000000-0000-0000-C000-000000000046" .
+
+:productionForecast a samm:Property ;
+   samm:preferredName "Production Forecast"@en ;
+   samm:description "Date of completion"@en ;
+   samm:characteristic samm-c:Timestamp ;
+   samm:exampleValue "2023-06-19T21:24:00+07:00"^^xsd:dateTime .
+
+:precisionOfForecast a samm:Property ;
+   samm:preferredName "Precision Of Forecast"@en ;
+   samm:description "Accuracy of the prediction"@en ;
+   samm:characteristic ext-types:TimeValueCharacteristic .
+
+:productionStatus a samm:Property ;
+   samm:preferredName "Production Status"@en ;
+   samm:description "Status of the order/position within MP"@en ;
+   samm:characteristic :ProductionStatusEnum ;
+   samm:exampleValue "itemPlanned" .
+
+:reasons4Delay a samm:Property ;
+   samm:preferredName "Reasons 4 Delay"@en ;
```

Comment:
Would this writing be the one the user sees? If so ok, if not adjust to maybe Reasons for Delay